coralproject/talk
# [CORL-1529] Add config option to pre-mod comments with suspect words

## What does this PR do?

Adds a pre-mod config option to make it so that comments with suspect words are flagged and put into pre-mod to be evaluated by a moderator.

## What changes to the GraphQL/Database Schema does this PR introduce?

Add `premoderateSuspectWords: Boolean` to the `Settings` and `SettingsInput`.

## How do I test this PR?

- Go to admin and enable the feature under `Configure > Moderation > Pre-moderation`.
- Add some suspect words to your suspect word list
- As a commenter, post a comment with one of the suspect words
- See that it is sent for moderation
- Check the mod queues and see the new comment sitting under Pending
